script/check-api-version

```diff
@@ -54,12 +54,15 @@ main()
         exit 0
     fi
 
-    git diff "${OT_SHA_OLD}" -- include/openthread | tee >(cat >&2) | grep -aP '[-+]#define OPENTHREAD_API_VERSION (.+)' >"${OT_VERSIONS_FILE}" || die 'Version number is not updated!'
+    git diff "${OT_SHA_OLD}" -- include/openthread | tee >(cat >&2) | grep '[-+]#define OPENTHREAD_API_VERSION' >"${OT_VERSIONS_FILE}" || die 'Version number is not updated!'
 
-    [[ $(wc -l <"${OT_VERSIONS_FILE}") == 2 ]] || die 'Multiple OPENTHREAD_API_VERSION definitions found!'
+    line_count="$(wc -l <${OT_VERSIONS_FILE})"
+    line_count="${line_count// /}"
```

Comment:
simpler?
```suggestion
    [[ $(($(wc -l <"${OT_VERSIONS_FILE}"))) == 2 ]] || die 'Multiple OPENTHREAD_API_VERSION definitions found!'
```

Reply:
yes, but the `[[` notation isn't portable and is why I did it in multiple steps